Plugins/Reqnroll.Microsoft.Extensions.DependencyInjection.ReqnrollPlugin/README.md

```diff
@@ -0,0 +1,13 @@
+﻿Reqnroll.Microsoft.Extensions.DependencyInjection provides [Microsoft.Extensions.DependencyInjection](https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-usage) integration for Reqnroll.
+
+This package enables Reqnroll to use the standard .NET dependency injection container for resolving step definitions, hooks, and other dependencies in your BDD tests.
```

Comment:
"standard .NET dependency injection container": I wouldn't call it "standard" (despite what MS thinks). I would say "This package enables Reqnroll to dependency injection container from the Microsoft.Extensions.DependencyInjection package for ..."